jobs/service-fabrik-broker/spec

```diff
@@ -52,7 +52,7 @@ provides:
   - monitoring.unauthorized.include_response_body
   - riemann.show_errors
   - riemann.log_additional_event
-  - riemann.skip_http_status_40x
+  - riemann.skip_http_status_response_codes
```

Comment:
This sounds more like a boolean variable name.

Can we name it like `http_status_codes_to_be_skipped` ?